ローカルで AtCoder のシステムテストを利用した verify ができない。

## Summary / 概要
AtCoder のテストケースでの verify を手元で行いたいのですが、環境変数に `DROPBOX_TOKEN` があってもエラーが出てしまいます。`oj download`によるダウンロードはできます。issue を立てること自体が初めてなため、情報の不足などがあるかと思われますが、その際はすみません、追記します。

## Steps to reproduce / 再現方法
申し訳ありません、未検証です。

environments:
- macOS BigSur(version 11.2.3)
- [リモートリポジトリ](https://github.com/atree-GitHub/competitive-library)
- fish shell
```console
$ fish -v
fish, version 3.2.2

$ oj --version
online-judge-tools 11.3.0 (+ online-judge-api-client 10.9.0)

$ pip3 list
Package                    Version
-------------------------- ---------
...
online-judge-api-client    10.9.0
online-judge-tools         11.3.0
online-judge-verify-helper 5.5.3
...

$ printenv
...
DROPBOX_TOKEN=(DROPBOX_TOKEN の値)
...

$ cat $XDG_CONFIG_HOME/fish/config.fish
...
set -x DROPBOX_TOKEN (DROPBOX_TOKEN)
...
```

## Expected behavior / 期待される挙動
AtCoder の DropBox からシステムテストケースがダウンロードされる。

## Actual behavior / 実際の挙動

``` console
$ oj-verify run lib/test/atcoder/abc110d.test.cpp
INFO:onlinejudge_verify.config:config file loaded: .verify-helper/config.toml: {'languages': {'cpp': {'environments': [{'CXX': 'g++', 'CXXFLAGS': ['-std=c++17', '-Wall', '-Wextra', '-g', '-fsanitize=undefined', '-D_GLIBCXX_DEBUG']}]}}}
WARNING:onlinejudge_verify.verify:failed to increase the stack size
::warning ::failed to ulimit
INFO:onlinejudge_verify.verify:verify: lib/test/atcoder/abc110d.test.cpp
INFO:onlinejudge_verify.verify:problem: https://atcoder.jp/contests/abc110/tasks/abc110_d
INFO:onlinejudge_verify.verify:$ sleep 2
INFO:onlinejudge_verify.verify:$ oj download --system -d .verify-helper/cache/8d80107966082795ebe032c622b73525/test --silent https://atcoder.jp/contests/abc110/tasks/abc110_d
[INFO] online-judge-tools 11.3.0 (+ online-judge-api-client 10.9.0)
[INFO] load cookie from: /Users/USERNAME/Library/Application Support/online-judge-tools/cookie.jar
[HINT] You need to give the access token. Please do the following:

    1. Open the following URL in your browser:
        https://www.dropbox.com/oauth2/authorize?client_id=153gig8dqgk3ujg&response_type=code
    2. Authorize the app and take the access code.
    3. Run the following command with replacing the "${YOUR_ACCESS_CODE}":
        $ curl https://api.dropbox.com/oauth2/token --user 153gig8dqgk3ujg:5l7o7lh73o8i9ux --data grant_type=authorization_code --data code=${YOUR_ACCESS_CODE}
    4. Get the access token from the JSON. It is in the "access_token" field.
    5. Use the access token. For example:
        $ oj download https://atcoder.jp/contests/abc110/tasks/abc110_d --system --dropbox-token=${YOUR_ACCESS_TOKEN}

(Please take care that the access code and the access token are CONFIDENTIAL information. DON'T SHARE with other people!)

[ERROR] --dropbox-token is not given
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/onlinejudge_command/main.py", line 101, in main
    sys.exit(run_program(parsed, parser=parser))
  File "/usr/local/lib/python3.9/site-packages/onlinejudge_command/main.py", line 61, in run_program
    if not subcommand_download.run(args):
  File "/usr/local/lib/python3.9/site-packages/onlinejudge_command/subcommand/download.py", line 112, in run
    raise SampleParseError("--dropbox-token is not given")
onlinejudge.type.SampleParseError: --dropbox-token is not given
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/onlinejudge_verify/verify.py", line 85, in verify_file
    exec_command(command)
  File "/usr/local/lib/python3.9/site-packages/onlinejudge_verify/verify.py", line 42, in exec_command
    subprocess.check_call(command)
  File "/usr/local/Cellar/python@3.9/3.9.4/Frameworks/Python.framework/Versions/3.9/lib/python3.9/subprocess.py", line 373, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['oj', 'download', '--system', '-d', '.verify-helper/cache/8d80107966082795ebe032c622b73525/test', '--silent', 'https://atcoder.jp/contests/abc110/tasks/abc110_d']' returned non-zero exit status 1.
::error file=lib/test/atcoder/abc110d.test.cpp::failed to verify
ERROR:onlinejudge_verify.verify:1 tests failed
ERROR:onlinejudge_verify.verify:failed: lib/test/atcoder/abc110d.test.cpp
```


## Other notes / その他
`oj download`によるインストールには成功しています。
```console
oj download https://atcoder.jp/contests/abc197/tasks/abc197_a --system --dropbox-token=$DROPBOX_TOKEN                                                                             Sat May 29 07:10:14 2021
[INFO] online-judge-tools 11.3.0 (+ online-judge-api-client 10.9.0)
[INFO] load cookie from: /Users/USERNAME/Library/Application Support/online-judge-tools/cookie.jar
[NETWORK] POST: https://api.dropboxapi.com/2/files/list_folder
[NETWORK] 200 OK
[INFO] contest folder found: {'.tag': 'folder', 'name': 'ARC120', 'parent_shared_folder_id': '1276086752', 'id': 'id:hjsYZQdnqVAAAAAAAAGKYg', 'sharing_info': {'read_only': True, 'parent_shared_folder_id': '1276086752', 'traverse_only': False, 'no_access': False}}
[NETWORK] POST: https://api.dropboxapi.com/2/files/list_folder
[NETWORK] 200 OK
[INFO] problem folder found: {'.tag': 'folder', 'name': 'F', 'parent_shared_folder_id': '1276086752', 'id': 'id:hjsYZQdnqVAAAAAAAAF9sw', 'sharing_info': {'read_only': True, 'parent_shared_folder_id': '1276086752', 'traverse_only': False, 'no_access': False}}
[NETWORK] POST: https://api.dropboxapi.com/2/files/list_folder
[NETWORK] 200 OK
[NETWORK] POST: https://content.dropboxapi.com/2/sharing/get_shared_link_file
[NETWORK] 200 OK
...
[INFO] save cookie to: /Users/USERNAME/Library/Application Support/online-judge-tools/cookie.jar
[INFO] append the downloading history: /Users/USERNAME/Library/Caches/online-judge-tools/download-history.jsonl

[INFO] sample 0
[INFO] input: handmade_00.txt
zzz

[SUCCESS] saved to: test/handmade_00.txt.in
[INFO] output: handmade_00.txt
zzz
...
```

-   Will you try to create a pull request?
    -  no

--- a/setup.py
+++ b/setup.py
@@ -3,7 +3,7 @@ from setuptools import find_packages, setup
 
 setup(
     name='online-judge-verify-helper',
-    version='5.5.3',
+    version='5.6.0',
     author='Kimiyuki Onaka',
     author_email='kimiyuki95@gmail.com',
     url='https://github.com/kmyk/online-judge-verify-helper',

